projects/joystick-servo-gimbal.md

```diff
@@ -1 +1,84 @@
-# Arduino Joystick and Servo Gimbal
\ No newline at end of file
+# Candy Dispenser
+
+## Introduction
+
+In this project, you will learn how to build a simple robotic arm using two servo motors, a joystick, and an Arduino UNO.  
+
+<div style="text-align:center">
+    <img src="assets/joystick-servo-gimbal.gif" width = 200>
```

Comment:
the assets folder doesn't exist in this branch, we should add it in a new commit before we merge this, otherwise this is a broken link